iothub/service/src/IotHubServiceClient.cs

```diff
@@ -142,7 +149,14 @@ public IotHubServiceClient(string hostName, AzureSasCredential credential, IotHu
         public ConfigurationsClient Configurations { get; protected set; }
 
         /// <summary>
-        /// Dispose this client and all the disposable resources it has. This includes any HTTP clients
+        /// Subclient of <see cref="IotHubServiceClient"/> that handles all digital twin operations including
+        /// getting a digital twin, updating a digital twin, and invoking commands on a digital twin.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/en-us/azure/iot-develop/concepts-digital-twin"/>
```

Comment:
Remove `en-us` part of URI.

```suggestion
        /// <seealso href="https://docs.microsoft.com/azure/iot-develop/concepts-digital-twin"/>
```